apps/web/src/app/(dashboard)/admin/stats/user-with-document-cummulative.tsx

```diff
@@ -0,0 +1,69 @@
+'use client';
+
+import { DateTime } from 'luxon';
+import { Bar, BarChart, ResponsiveContainer, Tooltip, XAxis, YAxis } from 'recharts';
+
+import type { GetUserWithDocumentMonthlyGrowth } from '@documenso/lib/server-only/admin/get-users-stats';
+
+export type UserWithDocumentCummulativeChartProps = {
+  className?: string;
+  data: GetUserWithDocumentMonthlyGrowth;
+};
+
+export const UserWithDocumentCummulativeChart = ({
+  className,
+  data,
+}: UserWithDocumentCummulativeChartProps) => {
+  const formattedData = [...data]
+    .reverse()
+    .map(({ month, cume_count: count, cume_signed_count: signed_count }) => {
+      return {
+        month: DateTime.fromFormat(month, 'yyyy-MM').toFormat('LLLL'),
```

Comment:
It looks like some months aren't showing up due to being cropped out or something

Can we use `LLL` instead for short months?